docs/css/custom.css

```
@@ -240,10 +240,13 @@ body {
 }
 
 .md-typeset code {
-    background-color: #f0f0f0;
+    background-color: #ededed;
     padding: 1px 4px;
     margin: 0;
-    box-shadow: none;
+    box-shadow: 0 1px 1px rgba(0, 0, 0, 0.125);
+    color: #f05b01;
```

```suggestion
    color: #2b2b2b;
```

We use a very similar orange color for links, so we can't have monospace in orange as well, but regular dark gray should be readable enough with your other suggestions (also passes WCAG 2 level AA)

---

Yeah I agree. #2b2b2b works well with monospace.